### PR TITLE
oadp-1.1: yq format when for `make test`. pin gomega to v1.22

### PR DESCRIPTION
**Pull request by kaovilai**
Cherrypick #846 to oadp-1.1

### CODE REVIEW

**Review by sseago — APPROVED** [review]

File: build/custom-ci-Dockerfile [hunk 1]
```diff
@@ -1,8 +1,9 @@
-FROM openshift/origin-release:golang-1.16
+FROM openshift/origin-release:golang-1.17
```
Comment (kaovilai) [inline build/custom-ci-Dockerfile:1]: note that this image tag `golang-1.17` doesn't actually exists at least in docker hub.

Will check and maybe remove this file if it's not used anywhere.

Comment (kaovilai) [reply build/custom-ci-Dockerfile:1]: Will delete in a follow up later.

Comment (kaovilai) [reply build/custom-ci-Dockerfile:1]: looks to be a leftover from ansible days

**Comment by weshayutin** [comment]
Do we want this change on OADP-1.1.1 ?

**Comment by kaovilai** [comment]
Let's punt for 1.1.2

**Review by dymurray — APPROVED** [review]

**Pull request merged**